reservaciones/src/navigation/NavigationStack.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { createStackNavigator } from '@react-navigation/stack';
+import { Home } from '../screens/Home';
+
+const Stack = createStackNavigator();
+
+export const NavigationStack = () => {
+    return(
+        <Stack.Navigator screenOptions={{headerShown: false}}>
+            <Stack.Screen name="Home" component={Home} />
+        </Stack.Navigator>
+    );
+}
```

Comment:
Check the indentation of your file (a spacing of 2 is recommended).

Add a blank line at the end.